restapi/admin_tenants_helper.go

```
@@ -196,15 +196,15 @@ func getKESConfiguration(ctx context.Context, clientSet K8sClientI, ns string, e
 	kesConfigurationSecretName := fmt.Sprintf("%s-kes-configuration", secretName)
 
 	kesConfiguration = &operator.KESConfig{
-		Image:    "minio/kes:v0.11.0",
+		Image:    KESImageVersion,
 		Replicas: 1,
 	}
 	// Using custom image for KES
 	if encryptionCfg.Image != "" {
 		kesConfiguration.Image = encryptionCfg.Image
 	}
-	// Generate server certificates for KES only if autoCert is disabled
-	if autoCert == nil || (autoCert != nil && !*autoCert) {
+	// Generate server certificates for KES
+	if encryptionCfg.Server != nil {
```

if this is nil should we return any error or nah?

---

`encryptionCfg.Server` are External certificates for KES, in case this is `nil` KES will still be deployed using `AutoCert` certificates